### PR TITLE
update swagger parser

### PR DESCRIPTION
**Pull request by zeitlinger**
update swagger parser, as the custom changes have been merged upstream

### CODE REVIEW

**Comment by zeitlinger** [comment]
:+1:

File: server/build.gradle.kts [hunk 1]
```diff
@@ -9,6 +9,10 @@ buildscript {
         gradlePluginPortal()
         maven("https://repo.spring.io/libs-release")
     }
+
+    extra.apply {
+        set("jackson.version", "2.10.1")
```
[the remark this answers is not in the document]
Comment (zeitlinger) [reply server/build.gradle.kts]: 1. for some reason `2.10.2` is not the latest everywhere, e.g. https://mvnrepository.com/artifact/com.fasterxml.jackson.core/jackson-annotations
2. this statement is actually picked up by spring, which you can try out by running `./gradlew :dependencies`
3. I don't really like this solution, happy to improve it

File: server/zally-rule-api/build.gradle.kts [hunk 2]
```diff
@@ -1,6 +1,5 @@
 dependencies {
-    compile("io.swagger.core.v3:swagger-models:2.0.9")
-    compile("io.swagger:swagger-models:1.5.23")
-    // 2.9+ is invalid for maven publish
-    compile("com.fasterxml.jackson.module:jackson-module-kotlin:2.9.8")
+    compile("io.swagger.core.v3:swagger-models:2.1.1")
+    compile("io.swagger:swagger-models:1.6.0")
+    compile("com.fasterxml.jackson.module:jackson-module-kotlin:2.10.1")
```
[the remark this answers is not in the document]
Comment (zeitlinger) [reply server/zally-rule-api/build.gradle.kts]: it's already the latest, https://mvnrepository.com/artifact/com.fasterxml.jackson.module/jackson-module-kotlin

Gradle: yes, maven is simpler in this regard (or I'm just more used to it...)

File: server/build.gradle.kts (hunk 1 above)
[the remark this answers is not in the document]
Comment (roxspring) [reply server/build.gradle.kts]: You're absolutely right! Affecting the spring dependency management is a little surprising... am half tempted to request an explanatory comment, but the effect is discoverable so am not sure.

[the remark this answers is not in the document]
Comment (zeitlinger) [reply server/build.gradle.kts]: I'm not sure myself - so I just added the comment.

**Comment by zeitlinger** [comment]
:+1:

**Comment by tkrop** [comment]
:+1:

**Comment by zeitlinger** [comment]
:+1:

**Pull request merged**